src/linux/minidump_writer.rs

```diff
@@ -203,14 +203,14 @@ impl MinidumpWriter {
             }
         }
 
-        let mut buffer = Cursor::new(Vec::new());
+        let mut buffer = Buffer::with_capacity(4 * 1024);
```

Comment:
No, this was just kind of arbitrary (~page size), the minimum would probably just be the MINIDUMP_HEADER, this was just to avoid unnecessary reallocs in the beginning but I can return it to empty or anything else.